site/overrides/home.html

```diff
@@ -262,9 +262,9 @@ <h2>
     <div class="feature-item">
         <a href="https://medium.com/mesh-for-data">
             <h2>
-            <center>Our blogs @ Medium.com</center>
+            <center>Read our blogs</center>
```

Comment:
I favor just "Blog" but if not then is it "blogs" or "blog posts"?

Reply:
Addressed.